docs/modules/general/pages/implementation-details/mfe-content-loading/webcomponents.adoc

```diff
@@ -0,0 +1,195 @@
+= Web Components in OneCX
+
+== Overview
+Web Components is a suite of different technologies used to create reusable custom elements with encapsulated functionality. In OneCX context they are usefull for dynamic content creation for both modules and Remote Components display of Microfrontends. Prior to reading this document it is recommended to read the documentation of xref:implementation-details/mfe-content-loading/index.adoc[microfrontend content loading] to get familiar with the content loading mechanisms available within the OneCX platform. 
+
+IMPORTANT: Webcomponent expose method enables the OneCX platform to use UI Apps developed with different frameworks and versions. It is the **recommended method for all Applications**.
+
+NOTE: Web Components technologies are used within OneCX platform, however OneCX offers its own functionalities that should be used to expose Microfrontend's content using the webcomponent method. Those functionalities are based on Web Components technologies, so there is no need to get familiar with them. Nevertheless, more resources about Web Components can be found https://developer.mozilla.org/en-US/docs/Web/API/Web_components[here]
+
+== Web Components preview
+Web Components are used in OneCX context for dynamic content creation. OneCX platform uses Web Components technologies to bound UI components to html elements. OneCX platform will use information from Application Store to determine what html elements have to be rendered to allow certain modules and Remote Components to bound to page content.
+
+With webcomponents method OneCX platform is only loading the code required for Microfrontend's content into the Shell UI memory and creating a "host" element for Remote Components or modules to attach into using Web Components. Microfrontend's exposed content is partially responsible for the content creation. Microfrontends have to adapt exposed modules and components so they are using Web Components Custom Elements.
+
+=== Custom Elements
+Web Components technology called Custom Elements is set of tools used to achive the bounding between components and html elements. As mentioned, OneCX platform will create html elements specified via configuration whenever a module or Remote Component has to be displayed on a page. There are 2 steps that OneCX platform takes when dealing with content loading with webcomponent expose method:
```

Comment:
... Custom Elements is **a** set of tools used to **achieve** the bounding between components and html elements.
... loading with **Webcomponent** expose method.